requirements.txt

```diff
@@ -8,4 +8,5 @@ python-glanceclient
 python-cinderclient
 python-neutronclient
 python-keystoneclient
+python-ironicclient
```

Comment:
So we have neutron/keystone/cinder things in Ursula's venv because of historical things we did, with calling these services on a cloud before doing a deploy into the cloud. In fact, we may be able to remove them. I believe almost all calls to cloud APIs are done _inside_ the newly built cloud now, instead of from the host system running Ansible.

I couldn't find anywhere in a quick search that makes Ironic client or library calls on the host system.